forte/version.py

```diff
@@ -19,3 +19,6 @@
 VERSION_SHORT = "{0}.{1}".format(_MAJOR, _MINOR)
 VERSION = "{0}.{1}.{2}".format(_MAJOR, _MINOR, _REVISION)
 FORTE_IR_VERSION = "0.0.1"
+PACK_VERSION = "0.0.1"
+DEFAULT_PACK_VERSION = "0.0.0"
+BACKWARD_COMPATIBLE_VER = "0.0.1"
```

Comment:
No point in this abbreviation
```suggestion
BACKWARD_COMPATIBLE_VERSION = "0.0.1"
```

Reply:
this is now renamed to PACK_ID_COMPATIBLE_VERSION, according to discussion/suggestion below. Where this will be put in will be discussed below.